WordPress/Classes/Stores/RemoteFeatureFlagStore.swift

```diff
@@ -0,0 +1,93 @@
+import Foundation
+import WordPressKit
+
+class RemoteFeatureFlagStore {
+
+    init() {
+        DDLogInfo("🚩 Remote Feature Flag Device ID: \(deviceID)")
+    }
+
+    /**
+     Fetches remote feature flags from the server.
+
+     - Parameter callback: An optional callback that can be used to update UI following the fetch. It is not called on the UI thread.
+    */
+    public func update(then callback: FetchCallback? = nil) {
+        // We currently use an anonymous connection to WordPress.com, because we don't need user state.
+        // In the future, we could provide login credentials to the endpoint, which would allow customizing flags server-side on a per-user basis.
+        let remote = FeatureFlagRemote(wordPressComRestApi: WordPressComRestApi.defaultApi())
+        remote.getRemoteFeatureFlags(forDeviceId: deviceID) { [weak self] result in
+            switch result {
+                case .success(let flags):
+                    self?.cache = flags.dictionaryValue
+                    DDLogInfo("🚩 Successfully updated local feature flags: \(flags)")
+                    callback?()
+                case .failure(let error):
+                    DDLogError("🚩 Unable to update Feature Flag Store: \(error.localizedDescription)")
+            }
+        }
+    }
+
+    /// Checks if the local cache has a value for a given `FeatureFlag`
+    public func hasRemoteValueForFlag(_ flag: FeatureFlag) -> Bool {
+        return flag.remoteKey != nil && cache[flag.remoteKey!] != nil
+    }
+
+    /// Looks up the value for a remote feature flag.
+    ///
+    /// If the flag exists in the local cache, the current value will be returned.  If the flag does not exist in the local cache, the compile-time default will be returned.
+    /// - Parameters:
+    ///     - flag: The `FeatureFlag` object associated with a remote feature flag
+    public func valueForFlag(_ flag: FeatureFlag) -> Bool {
```

Comment:
Same comment as above:

```swift
public func value(for flag: FeatureFlag) -> Bool {
```

Which would result in calls like this one:

```swift
remoteFeatureFlagStore.value(for: .newLoginFlow)
```

Reply:
Same as above!